floaters/src/main/java/com/mdg/droiders/floaters/CollapsedWindow.java

```diff
@@ -207,7 +207,7 @@ public boolean onTouch(View v, MotionEvent event) {
                         setInitialPos(params.x, params.y);
                         //get the touch location
                         setInitialTouchPos(event.getRawX(), event.getRawY());
-
+                        mClosingButtonView.setVisibility(View.VISIBLE);
```

Comment:
\[Checkstyle\] WARNING: 'block' child have incorrect indentation level 24, expected level should be one of the following: 16, 18, 20\.